components/logger.js

```diff
@@ -228,7 +228,7 @@ const WalletLoggerProvider = ({ children }) => {
     const log = { wallet, level, message, ts: +new Date() }
     saveLog(log)
     setLogs((prevLogs) => [...prevLogs, log])
-  }, [setLogs, saveLog])
+  }, [saveLog])
```

Comment:
State setters never change so they don't need  to be added to effect dependencies. Just wanted to make this dependency array consistent with other dependency arrays I am currently working on.

Probably too annoying to make it consistent everywhere (add state setters as dependencies or not). I think it's just opinion.